gulpfile.esm.js

```diff
@@ -97,7 +94,7 @@ function cordovaCompile() {
   const compileArgs = platform === "ios" ? "--device" : "";
   let releaseArgs = "";
   if (buildMode === "release" && platform === "android") {
-    releaseArgs = `--release -- --keystore=${KEYSTORE} --storePassword=${STOREPASS} --alias=${KEYALIAS} --password=${KEYPASS}`;
+    releaseArgs = `--release -- --keystore=keystore.p12 --alias=privatekey --storePassword=${androidKeyStorePassword} --password=${androidKeyStorePassword}`;
```

Comment:
No flags with secrets.

Reply:
How do i pass this into cordova, then? The alternative is a `build.json` - maybe we copy that from a siging repo? https://cordova.apache.org/docs/en/10.x/guide/platforms/android/#signing-an-app